app/(main)/login/page.test.tsx

```diff
@@ -33,10 +40,17 @@ jest.mock('../../../context/AuthContextProvider', () => ({
 }));
 
 describe('Login', () => {
+  const setIsLoading = jest.fn();
+  const useStateMock = (init: any): [unknown, Dispatch<unknown>] => [
+    init,
```

Comment:
Should it have the type of any and unknown? Could it have other type?

Reply:
@shashilo since you wrote this, can you explain?

Reply:
Unknown because the type could be anything.